cms/auth/middleware.py

```diff
@@ -32,6 +33,11 @@ def process_request(self, request: "HttpRequest") -> None:
         """
         super().process_request(request)
 
+        # Bypass authentication for internal health check endpoints.
+        if request.path in BYPASSED_INTERNAL_PATHS:
+            logger.debug("Bypassing authentication for internal health check request.", extra={"path": request.path})
```

Comment:
Issue: If there's logging, this doesn't really reduce the logger noise. I'd suggest dropping this entirely and letting the requests through silently.